packages/hash/frontend/src/components/layout/PageHeader/PageHeader.tsx

```diff
@@ -35,25 +50,60 @@ export const PageHeader: React.VFC = () => {
   ));
 
   return (
-    <header
-      className={tw`bg-white h-16 flex items-center border(b-1 gray-300)`}
-      style={{ paddingLeft: SIDEBAR_WIDTH + 10 }}
+    <Box
+      component="header"
+      sx={{
+        background: theme.palette.common.white,
+        borderBottom: `1px solid ${theme.palette.gray["30"]}`,
+        display: "flex",
+        alignItems: "center",
+        height: "4rem",
+      }}
     >
-      {user ? (
-        <Nav tw="justify-between">
-          <SearchBar />
-          <AccountDropdown name={user.properties.shortname!} logout={logout} />
-        </Nav>
-      ) : (
-        <Nav tw="justify-end">
-          <Link href="/signup">
-            <a className={tw`pb-0 border-b-0 hover:border-b-0`}>
-              <Button className="mr-3">Sign up</Button>
-            </a>
-          </Link>
-          <Button onClick={showLoginModal}>Sign in</Button>
-        </Nav>
-      )}
-    </header>
+      <Nav>
+        <Box
+          sx={{
+            display: "flex",
+            justifyContent: {
+              xs: "space-between",
+              md: "unset",
+            },
+            width: isMobile && user ? "100%" : undefined,
+          }}
+        >
+          <Box sx={{ display: "flex", alignItems: "center" }}>
+            <Link noLinkStyle href={`/${user ? user.accountId : ""}`}>
+              <HashNavIcon sx={{ height: theme.spacing(2), width: "auto" }} />
+            </Link>
+          </Box>
+          {user && <SearchBar />}
+        </Box>
+        {user ? (
+          <Box sx={{ display: "flex", alignItems: "center" }}>
+            <ActionsDropdown accountId={accountId} />
+            <NotificationsDropdown />
+            <AccountDropdown logout={logout} user={user} />
+          </Box>
+        ) : (
+          <Box>
+            <Button
+              variant="tertiary_quiet"
+              sx={{ mr: 1 }}
+              onClick={showLoginModal}
+            >
+              Log In
+            </Button>
+
+            <Button
+              size="small"
+              onClick={() => router.push("/signup")}
+              className="mr-3"
```

Comment:
Is this leftover tw styling? If so it's probably worth removing/replacing with MUI styling